episodes/episodes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Episodes
+teaching: 5
+exercises: 25
+---
+
+At the end of the previous section,
+you defined the learning objectives for your lesson as a whole.
+
+Rather than write the lesson as a single, long document,
+we recommend that you break it up into chunks
+like chapters in a book or episodes in a season of a TV series.
+This can help manage learners' cognitive load by 
+ensuring that you organise content into coherent, more self-contained chunks,
+and makes it easier for instructors to schedule regular and frequent breaks while teaching.
+Thinking about how content can be broken down like this
+early in the lesson design process
+helps you to consider the path learners will take to reach your defined objectives,
+and identify the component skills and knowledge they will need to pick up on the way.
+
+In The Carpentries, we refer to these individual parts of a lesson as _episodes_,
+to encourage lesson developers to think about them as 
+self-contained units (with their own narrative arc) 
+that nevertheless contribute to the larger whole
+(the theme or story that runs through a full season).
+It also helps to think about the typical length of an episode:
+these chunks should not take more than an hour to finish.
```

Comment:
I chose "finish" because I couldn't think of a better word that would encapsulate teaching+exercise time. Perhaps people will understand that time for exercises is implied in "teach", though?

Reply:
I like your recommendation to add a minimum length for episodes too, but I might go with 20-60 minutes?

Reply:
fixed in [4cc378e](https://github.com/carpentries/lesson-development-training/pull/231/commits/4cc378eb96e0e7c85768545ba5128a3999b0e35d)